src/clients/starknet-tx/constants.json

```diff
@@ -1,7 +1,7 @@
 {
   "authenticators": {
-    "vanilla": "0x6ad07205a4d725c5c2b10c4f5fbdfaaa351c742fce7a5a22b2b56fd8d5afd62",
-    "ethSig": "0x594a81b66c3aa2c64577916f727e1307b60c9d6afa80b6f5ca3e3049c40f643"
+    "0x6ad07205a4d725c5c2b10c4f5fbdfaaa351c742fce7a5a22b2b56fd8d5afd62": "vanilla",
+    "0x594a81b66c3aa2c64577916f727e1307b60c9d6afa80b6f5ca3e3049c40f643": "ethSig"
   },
   "strategies": {
```

Comment:
When strategies become dynamic the maping will be changed to address -> type too.